src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -26,12 +27,17 @@ def load_df_with_cache(
     feature_name: str,
 ) -> pd.DataFrame:
     """Wrapper function to cache dataframe loading."""
+    start_time = time.time()
     log.info(
         f"{feature_name}: Loading values",
     )  # pylint: disable=logging-fstring-interpolation
+
     df = loader_fn(**kwargs)
+
+    end_time = time.time()
+
     log.info(
-        f"{feature_name}: Loaded values",
+        f"{feature_name}: Loaded in {end_time - start_time:.2f} seconds",
```

Comment:
I'm afraid that we'll end up spamming the logs with lines like these which are only situationally useful. Perhaps reducing logging level to debug in this case would do the trick? adding verbosity flags would be painful and annoying..